services/SubmissionService.ts

```diff
@@ -1,10 +1,20 @@
 import axios from "axios";
+import { FormSchemaType } from "../pages/patient-information";
 import { useAuthStore } from "../src/store/auth";
 
 const api = "http://localhost/api";
 
 const token = useAuthStore.getState().token;
 
+export const completeProfile = async (data: FormSchemaType) => {
+  const res = await axios.post(`${api}/complete-profile`, data, {
```

Comment:
do you care about the response this endpoint gives? if so please put the response type here on the post function `.post<Type>(...)`